### DISCUSSION TITLE
How to  pass true false parent to child component?

### DISCUSSION THREAD

**Discussion by web-emran** [community/community#65418] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How to  pass true false parent to child component?

**Comment by coderwadud** [comment]
@Output() valueToParent: EventEmitter<boolean> = new EventEmitter<boolean>();

  sendValueToParent() {
    const booleanValue = true; // or false, depending on your case
    this.valueToParent.emit(booleanValue);
  }

**Comment by Akash1134** [comment]
Hi @coderwadud and @web-emran 

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
